src/containers/BooksForm.js

```diff
@@ -10,20 +13,53 @@ const CATEGORY = [
   'Sci-Fi',
 ];
 
-const BooksForm = () => (
-  <form>
-    <h2>Add book form</h2>
-    <input type="text" placeholder="Enter Book Title" />
-    <select name="category" id="category">
-      <option value="">Select a category</option>
-      {CATEGORY.map((category) => (
-        <option key={Math.floor(Math.random() * CATEGORY.length)} value={category}>
-          {category}
-        </option>
-      ))}
-    </select>
-    <input type="button" value="submit" />
-  </form>
-);
-
-export default BooksForm;
+const BooksForm = ({ addBook }) => {
+  const [titleInput, setTitleInput] = useState('');
+  const [categorySelect, setCategorySelect] = useState('');
+
+  const handleChange = (e) => {
+    if (e.target.name === 'title') {
+      setTitleInput(e.target.value);
+    } else {
+      setCategorySelect(e.target.value);
+    }
+  };
+
+  const handleSubmit = () => {
+    addBook({
+      ISBN: Math.floor(Math.random() * 100000001),
+      title: titleInput,
+      category: categorySelect,
+    });
+    setTitleInput('');
+    setCategorySelect('');
+  };
+
+  return (
+    <form>
```

Comment:
For the booksForm handleSubmit function, It has to be attached to the form's onSubmit event. This way you can add validation to the input fields by adding the required attribute, and write less code as it is also one of the best practices.
 
 Screenshot from pathwright:
 
![image](https://user-images.githubusercontent.com/31897434/127893627-9b03d3b3-5839-4288-be5c-2f6a42eb11a2.png)